src/Model/Item.php

```diff
@@ -3823,6 +3823,12 @@ private static function addLinkAttachment(int $uriid, array $attachments, string
 		} elseif (preg_match("/.*(\[attachment.*?\].*?\[\/attachment\]).*/ism", $body, $match)) {
 			$data = BBCode::getAttachmentData($match[1]);
 		}
+
+		if ($sensitive) {
```

Comment:
This should also be toggled by the user choice to blur sensitive images or not.

Reply:
It already is.

Reply:
Thank you for the clarification, because it isn't clear from the code.

Reply:
`$sensitive` is filled at the beginning of the whole routine vis the value of the post - but in combination with the user configuration. So when you disabled the behaviour, `$sensitive` will never be true.